libs/ts-rest/express/src/lib/ts-rest-express.ts

```diff
@@ -130,6 +131,14 @@ const validateRequest = (
   };
 };
 
+const getExpressPath: (path: string) => string =
+  // The app.del() method exists in Express v4 but was removed in v5
+  'del' in application
+    ? // Express v4 accepts ts-rest paths without modification
+      (path) => path
+    : // Express v5 requires {/:param} instead of /:param? for optional params
```

Comment:
When I first looked at the code I was confused because I thought ts-rest was sending the path directly to the integration, but in hindsight, there needs to be some ts-rest path convention that works for both clients + servers. I now understand why this switch is here. 

Separately, it would probably be good for the docs to be updated to explain a bit about how paths are supported and translated in contracts. Maybe it's there already and I completely missed it. 

Awesome that you got this to support both versions 🦸

Reply:
If anyone else was confused like I was, check out https://github.com/ts-rest/ts-rest/blob/main/libs/ts-rest/core/src/lib/paths.ts 😄

Reply:
This is great! I'll definitely play with this locally, possibly find a way to test this - anytime I see a mysterious regex I get mildly concerned haha